hazelcast/dataset/META-INF/CompilerHints

```diff
@@ -0,0 +1,5 @@
+dontinline,*.*_all_jmhStub
```

Comment:
What's this then? :)

Reply:
Ah good one :D

This is left over from the code generated by project-x. In my branch the dataset directory has been excluded; but in this branch it isn't.